src/site/resources/css/site.css

```diff
@@ -98,10 +98,10 @@ p, td, table.bodyTable td, th, table.bodyTable th, li, pre, #breadcrumbs span, #
 }
 
 #leftColumn {
-  width: 270px;
+  min-width: 270px;
+  width: fit-content;
```

Comment:
Min width on my laptop is 250. If we are going to do this, we may as well decrease this.